charts/jaeger/values.yaml

```diff
@@ -90,11 +90,8 @@ cassandra:
 
 # Begin: Override values on the Kafka subchart to customize for Jaeger
 kafka:
-  replicas: 1
-  configurationOverrides:
-    "auto.create.topics.enable": true
-  zookeeper:
-    replicaCount: 1
+  autoCreateTopicsEnable: true
+
```

Comment:
by default, kafka and zookeeper replica values are 1 for bitnami chart.

Reply:
One change needed here. Although default replica set will be 1 for `kafka` and `zookeeper` but It would be better to set it explicitly. When any user wants to change this value, he/she wouldn't have to go to Bitnami's Documentation and find correct `key`

Reply:
Sure. Although one could make that same argument for few other key's 😄 . I ll push that change up.